### PR TITLE
UI Initial add sources wizard

### PR DESCRIPTION
**Pull request by cdcabrera**
Initial wizard steps added for updating sources, should only display basic sample copy. Since it affects the shared ```components/sources/sources.js``` file, checking it in early.

Need an opinion about where to place this component next round? Either leave it under -
- ```components/sources/..``` 

or place it under it's own  directiory like 
- ```components/addSourceWizard/..```

updates #534

### CODE REVIEW

File: client/src/components/sources/sources.js [hunk 1]
```diff
@@ -148,13 +152,21 @@ class Sources extends React.Component {
...
       type: toastNotificationTypes.TOAST_ADD,
       alertType: 'error',
       header: 'NYI',
       message: 'Adding sources is not yet implemented'
     });
+    */
+  }
+
+  hideAddSourceWizard() {
+    this.setState({ addSourceWizardShown: false });
```
Comment (jeff-phillips-18) [inline client/src/components/sources/sources.js]: We should probably ask for confirmation so an inadvertent click outside the wizard doesn't dismiss and force the user to start over again.

File: client/src/components/sources/sources.js [hunk 2]
```diff
@@ -148,13 +152,37 @@ class Sources extends React.Component {
...
+      Store.dispatch({
+        type: confirmationModalTypes.CONFIRMATION_MODAL_HIDE
+      });
+
+      this.setState({ addSourceWizardShown: false });
+    };
+
+    Store.dispatch({
+      type: confirmationModalTypes.CONFIRMATION_MODAL_SHOW,
+      title: 'Quit Adding Source',
```
Comment (jeff-phillips-18) [inline client/src/components/sources/sources.js]: How about 'Cancel Add Source'?

File: client/src/components/sources/sources.js [hunk 3]
```diff
@@ -148,13 +152,37 @@ class Sources extends React.Component {
...
       type: toastNotificationTypes.TOAST_ADD,
       alertType: 'error',
       header: 'NYI',
       message: 'Adding sources is not yet implemented'
     });
+    */
+  }
+
+  quitAddSourceWizard() {
+    let heading = <h3>Are you sure you want to quit adding a source?</h3>;
```
Comment (jeff-phillips-18) [inline client/src/components/sources/sources.js]: cancel rather than quit

File: client/src/components/sources/sources.js (hunk 2 above)
Comment (cdcabrera) [reply client/src/components/sources/sources.js]: heh so i started to use the ```cancel``` wording except... there's a "cancel" button on the confirmation modal... that feels like a dark pattern...

File: client/src/components/sources/sources.js (hunk 3 above)
Comment (cdcabrera) [reply client/src/components/sources/sources.js]: I'd love to use the "cancel" wording, but unless we change the name of the "cancel" button, I think we should be careful on that one.

Comment (cdcabrera) [reply client/src/components/sources/sources.js]: Per @serenamarie125 @bclarhk we're aiming to use ```no``` and ```yes``` for the buttons

**Pull request merged**